app/src/main/res/layout/fragment_relation.xml

```diff
@@ -24,4 +24,20 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/tlMentorshipRelation" />
 
+    <android.support.v7.widget.AppCompatTextView
+        android:id="@+id/tvNoCurrentRelation"
+        android:layout_width="250dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="@string/no_current_mentorship_relation"
+        android:textAlignment="center"
+        android:textColor="@android:color/black"
```

Comment:
You didn't remove this line that reinforces black color on the TextView. Please remove that

Reply:
can you remove this line as I requested, please @mohak1283 ?